libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -438,13 +449,13 @@ const ParentalLeaveTemplate: ApplicationTemplate<
                   Promise.resolve(val.InReview),
                 ),
               read: 'all',
-              write: 'all',
             },
           ],
         },
-        on: {
-          [DefaultEvents.EDIT]: { target: States.EDIT_OR_ADD_PERIODS },
-        },
+        // TODO: enable for user to do changes after application got APPROVE from VMST?
```

Comment:
Should this not be addressed before merging for main? As everything going on main should be ready for production.